controllers/rhmi/bootstrapReconciler.go

```diff
@@ -858,3 +868,39 @@ func getSecretQuotaParam(installation *rhmiv1alpha1.RHMI, serverClient k8sclient
 
 	return "", fmt.Errorf("waiting for quota parameter for 1 minute after creation of cr")
 }
+
+func (r *Reconciler) createAutoscalingConfigMap(ctx context.Context, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+	autoscalingConfig := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "autoscaling-config",
+			Namespace: r.installation.Namespace,
+		},
+	}
+
+	err := serverClient.Get(ctx, k8sclient.ObjectKey{
+		Name:      autoscalingConfig.ObjectMeta.Name,
+		Namespace: r.installation.Namespace,
+	}, autoscalingConfig)
+
+	// If configmap doesn't exist - i.e. an error is received, create and populate it
+	if err != nil && k8serr.IsNotFound(err) {
+		if autoscalingConfig.Data == nil {
+			autoscalingConfig.Data = map[string]string{}
+		}
+
+		autoscalingConfig.Data["backend-listener"] = "40"
+		autoscalingConfig.Data["backend-worker"] = "60"
+		autoscalingConfig.Data["apicast-production"] = "40"
+		autoscalingConfig.Data["ratelimit"] = "40"
+		autoscalingConfig.Data["usersso"] = "40"
+
+		err := serverClient.Create(ctx, autoscalingConfig)
```

Comment:
We should be using a createOrUpdate function. I seen one test case is checking that the values don't get reconciled. What is the reason for wanting this behaviour?